test/helpers/Fixtures.sol

```diff
@@ -476,6 +476,7 @@ contract DeploymentFixtures is StdCheats, DeploymentHelpers {
     function getDepositableNodeOperator(
         address nodeOperatorAddress
     ) internal returns (uint256 noId, uint256 keysCount) {
+        csm.cleanDepositQueue(100);
```

Comment:
I think it's better to skip invalid batches here where depositable < keys in batch